.travis.yml

```diff
@@ -1,10 +1,21 @@
-language: python
+#See https://github.com/travis-ci/travis-ci/issues/2312 for info on why we need
+# to install python manually
+# Python isn't supported OOTB, so pick a language that makes Travis happy
+language: objective-c
+os: osx
```

Comment:
This script assumes OSX - it uses brew. Would be nice to support multiple environments, but my understanding was we were limiting to OSX to start. Is that correct @gsfr?